tests/base/test_tests.py

```diff
@@ -0,0 +1,67 @@
+import pytest
+import discretize
+import numpy as np
+from discretize.tests import dottest
+from discretize.utils import volume_average
+
+
+class TestDottest:
+    def test_defaults(self, capsys):
+        # Use volume_average to test default case.
+
+        h = [[11, 20.5, 33], [222, 111, 333], np.ones(4) * 55.5]
+        mesh1 = discretize.TensorMesh(h, "CCC")
+
+        hx = np.ones(20) * 20
+        mesh2 = discretize.TensorMesh([hx, hx, hx], "CCC")
+
+        P = discretize.utils.volume_average(mesh1, mesh2)
+
+        # return
+        assert dottest(
```

Comment:
if dottest raises the proper assertions (when switching to `np.testing.assert_allclose`, then we do not need to `assert` the dot test.

Reply:
Done.